frameworks/kafka/docs/upgrade.md

```diff
@@ -0,0 +1,111 @@
+---
+post_title: Upgrade Kafka from 0.10.2.1 to 0.11.0.0
+menu_order: 10
+feature_maturity: preview
+enterprise: 'no'
+---
+
+# Upgrade Kafka from 0.10.2.1 to 0.11.0.0
+
+
+This document explains how to upgrade the DC/OS Kafka service from 0.10.2.x to 0.11.0.0 version. Refer to the Kafka [upgrade](https://kafka.apache.org/documentation/#upgrade) documentation for more information.
+
+- Information about the new message format in 0.11.0.0 is available [here](https://kafka.apache.org/documentation/#upgrade_11_message_format).
+
+- Kafka 0.11.0.0 introduces idempotent and transactional capabilities. These new 0.11.0.0 features require the new messaging format, and so will not work on older formats. 
+
+- 0.11.0.0 clients can communicate with older brokers, such as 0.10.2.x brokers. However 0.11.0.0 clients are required for these new features. 
+
+The rolling [upgrade procedure](https://kafka.apache.org/documentation/#upgrade) (from 0.10.2.x to 0.11.0.0) is explained below. This procedure guarantees no downtime during the upgrade.
+
+
+1. Upgrade the code to the new version:
```

Comment:
Step 1 is not the process the user need to follow with DC/OS. The user will be confused. Instead I would just sate ~ "Update to the latest version of the DC/OS Kafka Service". Since we are describing the process here in this doc, I would drop the link to the official Apache Kafka Upgrade docs so the user does not read those and then get confused about how to upgrade Kafka on DC/OS.